superset/connectors/sqla/views.py

```diff
@@ -213,8 +216,10 @@ class TableModelView(DatasourceModelView, DeleteMixin, YamlExportMixin):  # noqa
             "Whether to populate the filter's dropdown in the explore "
             "view's filter section with a list of distinct values fetched "
             'from the backend on the fly'),
+        'is_sqllab_view': _(
```

Comment:
Is there anyway to infer this rather than it be a form option?

Reply:
I could remove it from the `edit_columns` and have it only in the `show_columns` if you think it's better